test_runner/regress/test_installed_extensions.py

```diff
@@ -85,3 +87,26 @@ def test_installed_extensions(neon_simple_env: NeonEnv):
             assert ext["n_databases"] == 2
             ext["versions"].sort()
             assert ext["versions"] == ["1.2", "1.3"]
+
+    # check that /metrics endpoint is available
+    # ensure that we see the metric before and after restart
+    res = client.metrics()
+    info("Metrics: %s", res)
+    m = parse_metrics(res)
+    neon_m = m.query_all("installed_extensions", {"extension_name": "neon", "versions": "1.2,1.3"})
```

Comment:
Is it possible to use an array for the versions?

Reply:
this is a prometheus label, which is always a string (unless I am missing something).

so I don't think it would make any difference, `versions.split(',')` will return an array